index.js

```diff
@@ -73,10 +73,16 @@ function Slideout(options) {
   this._fx = options.fx || 'ease';
   this._duration = parseInt(options.duration, 10) || 300;
   this._tolerance = parseInt(options.tolerance, 10) || 70;
-  this._padding = parseInt(options.padding, 10) || 256;
+  this._padding = this._translateTo = parseInt(options.padding, 10) || 256;
+  this._side = options.side || 'left';
```

Comment:
Instead of `this._side` i'll go for `this._orientation` (here, on the internal representation, not at API level).

So I'll change 

`this._side = options.side || 'left';` into `this._orientation = options.side === 'right' ? -1 : 1;`

This will be helpful for the necessary math below.